python/README.md

```diff
@@ -54,6 +54,10 @@ JSONTestRunner().run(suite)
 
 ```
 
+This will load tests from the `tests/` directory in your autograder source code,
+[loading only files starting with `test` by default](https://docs.python.org/3/library/unittest.html#unittest.TestLoader.discover).
+`JSONTestRunner` included in `gradescope-utils`, as described below.
```

Comment:
not sure what this means, is it missing an "is"?
```suggestion
`JSONTestRunner` is included in `gradescope-utils`, as described below.
```

Reply:
oof, that's what I get for not proofreading a "simple" change. thanks!